mkinitramfs-ll.sh

```diff
@@ -356,10 +356,20 @@ for bin in dmraid mdadm zfs; do
 done
 module_group="${module_group/mdadm/raid}"
 
```

Comment:
Libraries should not be copied like this because every binary is copied into the initramfs with its library dependencies. See dobin() function in mkinitramfs-ll.{ba,z,}sh. So I will revert this commit unless necessary.